src/views/About.vue

```diff
@@ -76,6 +76,11 @@ export default Vue.extend({
         name: "杨书",
         desc: "22届后端（伪）",
       },
+      {
+        alias: "Lazenander",
+        name: "杨淏然",
+        desc: "23届打工人",
+      }
```

Comment:
As a hint, you forgot to add a comma here. Our lint rule requires a trailing comma for all arrays and objects.

But don't fix it by hand—let the linter fix it for you.